.devcontainer/initialize_command

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Maybe add a comment and link to to something like `eclipse-score/devcontainer/initialize_command.md`? For explanation of file selection magic.